### ISSUE TITLE
The real learning rate do not match with paper

### ISSUE THREAD

**Issue by what-is-available-for-name** [QitaoZhao/ContextAware-PoseFormer#12]
When I run your code step by step, I found the real learning rate in optimizer is 0.00016 because the second dictionary is empty in
 
(https://github.com/QitaoZhao/ContextAware-PoseFormer/blob/a2456578e8cd25f9fd99dacdf81d2e3623ca127b/ContextPose/train.py#L420C4-L425C6)

Thus all parameters are updated with learning rate 0.00016.

**Comment by what-is-available-for-name** [comment]
Maybe changing the parameters _"sampling_offsets"_ in function _match_name_keyword_ into _["sampling_offsets"]_ is what you really want.

Anyway, your model performs really excellent and impressive.

**Comment by QitaoZhao** [comment]
Thanks for pointing this out! I did not really notice that. Now, I am not able to compare the current behavior with the desired behavior (sampling offsets are expected to learn more slowly). The paper will be updated once I can do so.

Leave the issue open now.

**Comment by QitaoZhao** [comment]
I tried lowering the LR for deformable sampling parts (updated in https://github.com/QitaoZhao/ContextAware-PoseFormer/blob/main/ContextPose/train.py#L416-L428). The results seemed not quite different from our inadvertent original implementation (same LR for all). I may make the latter our default setting to align it with our checkpoints.

Again, thanks for letting me know this.
